### PR TITLE
Implement pym.js for embed + Outstanding changes

### PR DESCRIPTION
**Pull request by andela-mabdussalam**
#### What does this PR do?
This PR implements pym.js for the twitter embed. 

#### Description of Task to be completed?
Pym.js is implemented for the twitter embed.

#### Other Outstanding changes include- 

- Minor fix for deploy script.

- Edit for twitter summary card in meta tags.

- Style changes.

- Repositioning of logo in footer.

- Fix for CORS error.

- Fix for space in twitter link.

- Removing additional info from calculator container.

- Removing logo.

#### How should this be manually tested?

 Clone the repo and run with npm start.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -45,7 +49,7 @@
     ga('create', 'UA-93090833-2', 'auto');
     ga('send', 'pageview');
   </script>
-  
+  <script type="text/javascript" src="https://pym.nprapps.org/pym-loader.v1.min.js"></script>
```
Comment (DavidLemayian) [inline public/index.html]: Let's pull this from CDN.JS as it has better latency on the continent.

Comment (andela-mabdussalam) [inline public/index.html]: I am using the pym.js loader [here](http://blog.apps.npr.org/pym.js/#loader), which is not available on cdn.js. The pym.js script does not get loaded in the react component without the loader.

Comment (DavidLemayian) [inline public/index.html]: @andela-mabdussalam Hmm.. I think it is - https://cdnjs.com/libraries/pym

**Comment by DavidLemayian** [comment]
I've just realised that this is the Twitter embed that you mentioned. I'll close it since clarification.

**Pull request closed without merging**